src/components/DataViewFilterMultiselect/DataViewFilterMultiselectTypes.ts

```diff
@@ -27,4 +28,10 @@ export interface DataViewFilterMultiselectDropdownContentProps {
 	onApply: (val: any) => void;
 	isOpen: boolean;
 	placeholder?: DataViewFilterMultiselectProps["args"]["placeholder"];
+	limit?:  DataViewFilterMultiselectProps["args"]["limit"];
+	onChange?: (val: any) => void;
+	hideButtons?: boolean;
+	createNewOption?: (val: any) => void;
+	disabled?: boolean;
```

Comment:
This doesn't make much sense, if the AdvancedSelectionField is disabled then the button is disabled and there's no way for users to open the drawer.